packages/ton-client/src/shaders/basti-test.fs

```diff
@@ -0,0 +1,142 @@
+#version 300 es
```

Comment:
if I see it correctly, this shader file is not being used anywhere. 
Do we need it on main then?

Reply:
added it by accident, was just for me to test, will remove it agin